Add textbox for the grabbed links and encode links

wrick17 opened the pull request:
Added a textbox below the status text so that the user can still copy the grabbed links even if he overrides the clipboard. Should be the the fix for #8

Encoded the grabbed links, otherwise it fails to be recognized as links for some animes such as **TOKYO GHOUL √A**. Now they should work fine for anime with special characters in their names.

lap00zza commented:
Hi @wrick17 thank you so much for the PR. I just have one request. Can you add a hide button for the text box. Once people are done copying they probably wont want the textarea around.

wrick17 commented:
@lap00zza sure thing. I'll add that right away and update the PR.

wrick17 commented:
@lap00zza I have added the button. However, the status text that I have modified before seems to be too long and the text don't fit in one line anymore. It kinda bothers me. Maybe you can find a more appropriate text that gives the user the message, and fit properly 😁

lap00zza commented:
@wrick17 thanks again for the PR 😄
